src/styles/block-header/nav.scss

```diff
@@ -0,0 +1,58 @@
+.nav {
+  display: flex;
+  margin: 0 50px 0 0;
```

Comment:
it is better to set padding to header instead of setting margin here. especially considering the fact that we shouldn't set outer positioning for BEM block.